packages/extension-ui/src/components/Address.tsx

```diff
@@ -306,7 +306,7 @@ function Address ({ actions, address, children, className, genesisHash, isExtern
 }
 
 export default styled(Address)(({ theme }: ThemeProps) => `
-  background: ${theme.accountBackground};
+  background: ${theme.boxBackground};
```

Comment:
Since it is only used in accounts, not sure the rename is needed? (It did refer to location)

Reply:
I also used this for the signing data box background. Do you prefer to have different theme variables for account and box or just having a generic box background to be used for all type of these boxes makes sense?!